packages/expo/src/Expo.fx.tsx

```diff
@@ -22,7 +22,17 @@ const isManagedEnvironment =
 
 // If expo-font is installed and the style preprocessor is available, use it to parse fonts.
 if (StyleSheet.setStyleAttributePreprocessor) {
-  StyleSheet.setStyleAttributePreprocessor('fontFamily', Font.processFontFamily);
+  if (__DEV__) {
+    // Temporarily disable console.warn() in dev mode,
+    // because the experimented `StyleSheet.setStyleAttributePreprocessor` will show a warning about
+    // `Overwriting fontFamily style attribute preprocessor`.
+    const originalConsoleWarn = global.console.warn;
+    global.console.warn = () => {};
+    StyleSheet.setStyleAttributePreprocessor('fontFamily', Font.processFontFamily);
+    global.console.warn = originalConsoleWarn;
```

Comment:
Seems fine, let me know if this changes the stack traces though. If it does we'll need to add a file matcher to [this](https://github.com/expo/expo-cli/blob/ae23775587df6b4c69967b3e49e8f6d0d95d489b/packages/metro-config/src/ExpoMetroConfig.ts#L20).